src/scripts/Game.js

```diff
@@ -31,7 +32,7 @@ class Game {
     if (this.usedSurveyIds.length === 2) {
       startPanicRound(surveyObject);
     } else {
-      let round = new Round(surveyObject);
+      let round = new Round(surveyObject, this.players);
```

Comment:
If we get our class inheritance working we can get rid of the this.players argument in our new Round